python/ray/tests/test_failure.py

```diff
@@ -165,7 +166,7 @@ def get_val(self):
             return 1
 
     # There should be no errors yet.
-    assert len(ray.errors()) == 0
```

Comment:
this change is not needed right?

Reply:
You're right, I reverted the changes to this file.